styles.css

```diff
@@ -3,17 +3,18 @@
 * {
   margin: 0;
   padding: 0;
-  box-sizing: border-box;
+  /* box-sizing: border-box; */
   font-family: 'Poppins', sans-serif;
 }
 
 main {
-  position: absolute;
+  position: static;
   width: 375px;
   height: 5410px;
   background: #fff;
   margin-left: 30px;
   display: grid;
+  /* padding: 114px -35px 0 -1px; */
```

Comment:
- [ ] [REQURED] Please, remove commented code. Because it polluted the codebase. It's better to use `git reset` to come back to previous versions ◀️.